api/src/main/java/ai/djl/repository/zoo/Criteria.java

```diff
@@ -627,7 +627,7 @@ public Builder<I, O> optArgument(String key, Object value) {
          * @param options the model loading options
          * @return this {@code Builder}
          */
-        public Builder<I, O> optOptions(Map<String, String> options) {
+        public Builder<I, O> optOption(Map<String, String> options) {
```

Comment:
This should be `optOptions()`, this passing a map of options,
`optOption()` set one option at a time

Reply:
ok